File: seeker/snippet/indent.go
//date: 2021-12-27T16:42:38Z
//url: https://api.github.com/gists/f7a360d8d764042cfc5bda0de9f61172
//owner: https://api.github.com/users/dmitruk-v

package main

import (
	"fmt"
	"strings"
)

func main() {
	for i := 0; i < 10; i++ {
		fmt.Printf("%s\n", indent("- ", i))
	}
}

func indent(spacer string, level int) string {
	sb := strings.Builder{}
	for i := 0; i < level; i++ {
		sb.WriteString(spacer)
	}
	return sb.String()
}

File: seeker/snippet/indent.java
//date: 2022-08-30T16:49:31Z
//url: https://api.github.com/gists/f60d1467a4fd69545ce448851a82db41
//owner: https://api.github.com/users/halitgorgulu

public class IndentExample  
{  
    public static void main(String args[])   
    {  
           String text = "     Hello\n          from\n    otherside";  
           //değişmeden kalır
           System.out.println(text.indent(0));  
           //n tane boşluk silinir  
           System.out.println(text.indent(-6));  
           //n tane boşluk eklenir  
           System.out.println(text.indent(9));  
    }  
}
/*
     Hello
          from
    otherside

Hello
    from
otherside

              Hello
                   from
             otherside

*/